iml-gui/crate/src/components/font_awesome.rs

```diff
@@ -27,3 +27,34 @@ fn font_awesome_base<T>(sprite_sheet: &str, more_attrs: Attrs, icon_name: &str)
         ]
     ]
 }
+
+// cool symbols
+pub fn font_awesome_minus_circle<T>(awesome_style: Attrs) -> Node<T> {
```

Comment:
Why do we inline these instead of calling the correct thing from the svg sheet?

Reply:
Because these icons are from the _light_ set, which is not present in the fonts we use. 

I tried `plus-square` and `minus-square` from the _regular_ set:
![image](https://user-images.githubusercontent.com/940363/80575099-cbbf7780-8a24-11ea-87ad-10a28f7c522a.png)

`minus-circle` and `plus-circle` from the _solid_ set:
![image](https://user-images.githubusercontent.com/940363/80575371-47212900-8a25-11ea-935d-98fd195b2751.png)

These (hardcoded) icons:
![image](https://user-images.githubusercontent.com/940363/80575549-98311d00-8a25-11ea-9c94-93485cc2c3da.png)

For me, the latter ones are the clear winners. 😊

Reply:
How about we use the first ones, and make them a gray color.

I don't think we should inline the svg icons into our app.

Reply:
Done